### PR TITLE
add custom easyblock for code-server

### PR DESCRIPTION
**Pull request by ocaisa**

### CODE REVIEW

File: easybuild/easyblocks/c/code_server.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            mapped_arch = 'amd64'
+        elif myarch == AARCH64:
+            mapped_arch = 'arm64'
+        else:
+            raise EasyBuildError("Architecture %s is not supported for code-server on EasyBuild", myarch)
+
+        super(EB_code_minus_server, self).__init__(*args, **kwargs)
+
+        self.cfg.template_values['mapped_arch'] = mapped_arch
+        self.cfg.generate_template_values()
```
Comment (boegel) [inline easybuild/easyblocks/c/code_server.py:53]: Also implement the installation procedure and sanity check in the easyblock?

See also https://github.com/easybuilders/easybuild-easyconfigs/pull/11778#pullrequestreview-541684454

**Review by boegel — DISMISSED** [review]

File: easybuild/easyblocks/c/code_server.py (hunk 1 above)
Comment (ocaisa) [reply easybuild/easyblocks/c/code_server.py:54]: Done

**Comment by Micket** [comment]
Hm, what's up with ` easybuild/easyblocks/c/chimera.py` the empty file?

**Comment by ocaisa** [comment]
When I was in that dir I noticed it had execute permissions, so I removed them

**Review by Micket — APPROVED** [review]
lgtm

**Pull request merged**